src/components/inspectors/CycleManager.js

```diff
@@ -0,0 +1,100 @@
+import { DateTime } from 'luxon';
+import { getIso8601FormattedDateString, getPeriod } from './TimeUtils';
+
+export default class CycleManager {
+  _startDate;
+  _repeat;
+  _periodicityValue;
+  _selectedWeekdays;
+  _endDate;
+  _ends;
+  _times;
```

Comment:
Instead of using `_` to denote private properties, we've used `#` in other classes (which will turn it into a "real" private property). Do you think that'll help here?